packages/neptune-extension/src/labextension/components/ToolbarButton.less

```diff
@@ -1,11 +1,12 @@
-.n-toolbar-button {
-  min-width: 27px !important;
 
-  &--logo {
-    background-image: url('../../common/assets/logo.svg') !important;
-    background-size: 15px !important;
-    background-position: 6px center !important;
-    background-repeat: no-repeat !important;
-    padding-left: 23px !important;
-  }
+.neptune-icon {
+  background-image: url('../../common/assets/logo.svg') !important;
+  background-size: 15px !important;
+  background-position: 2px center !important;
+  background-repeat: no-repeat !important;
 }
+
+.status-icon-green {
```

Comment:
IMHO it is wrong, that you add style to icons to component file, and they are defined outside this component.